src/index.ts

```diff
@@ -353,6 +353,11 @@ function concatSig(v: Buffer, r: Buffer, s: Buffer): string {
   return ethUtil.addHexPrefix(rStr.concat(sStr, vStr)).toString('hex');
 }
 
+/**
+ * Normalize the input to a 0x-prefixed hex string.
+ *
+ * @param input - The value to normalize
```

Comment:
The return value is missing here

Reply:
Resolved in https://github.com/MetaMask/eth-sig-util/pull/178/commits/038f8476225ed1fd0938375ab14a51d9b51f81d0